messaging/src/main/java/org/axonframework/deadline/quartz/QuartzDeadlineManager.java

```diff
@@ -301,8 +320,20 @@ public Builder serializer(Serializer serializer) {
         }
 
         /**
-         * Sets a {@link Predicate} taking a {@link Throwable} to decided whether a failed {@link DeadlineJob} should
-         * be 'refired' immediately. Defaults to a Predicate which will refire immediately on
+         * Sets the {@link SpanFactory} implementation to use for providing tracing capabilities.
```

Comment:
Note that the `builder` methods throughout also define the default values. Hence, I'd expect all the infra components where you have added this default field to have a change in the `builder`'s JavaDoc too.